src/mapblock.h

```diff
@@ -213,17 +213,42 @@ class MapBlock /*: public NodeContainer*/
 		raiseModified(MOD_STATE_WRITE_NEEDED, MOD_REASON_SET_IS_UNDERGROUND);
 	}
 
-	inline void setLightingExpired(bool expired)
+	inline void setLightingIncomplete(u16 newflags)
 	{
-		if (expired != m_lighting_expired){
-			m_lighting_expired = expired;
-			raiseModified(MOD_STATE_WRITE_NEEDED, MOD_REASON_SET_LIGHTING_EXPIRED);
+		if (newflags != m_lighting_incomplete){
+			m_lighting_incomplete = newflags;
+			raiseModified(MOD_STATE_WRITE_NEEDED, MOD_REASON_SET_LIGHTING_INCOMPLETE);
 		}
 	}
 
-	inline bool getLightingExpired()
+	inline u16 getLightingIncomplete()
 	{
-		return m_lighting_expired;
+		return m_lighting_incomplete;
```

Comment:
I'm not 100% sure I like this -- it's almost like a double negative. Personally I'd prefer this function was called isLightingComplete() and the member called m_lighting_complete